data/sql_setup/prepare_schedule_e.sql

```diff
@@ -20,8 +20,54 @@ select *,
     now() as pg_date
 from fec_sched_e_notice_vw;
 
-update ofec_sched_e_tmp
```

Comment:
Woops...I think when you resolved the conflicts these two lines got deleted.  I cann add these back in.